asg.tf

```diff
@@ -3,9 +3,9 @@ resource "aws_launch_configuration" "launch_configuration" {
   name_prefix          = "${var.tags["Environment"]}-${var.tags["Application"]}-${var.tags["Tier"]}-${var.name}-LC/"
   image_id             = "${var.lc_ami_id}"
   instance_type        = "${var.lc_instance_type}"
+  key_name             = "${var.lc_key_name}"
```

Comment:
Are we not packering the keys we want to use and therefore don't want to add additional keys on launch?